src/jaegertracing/Tracer.h

```diff
@@ -194,6 +194,11 @@ class Tracer : public opentracing::Tracer,
 
     void close() noexcept { Close(); }
 
+    void Flush()
```

Comment:
I only use the title case functions for opentracing-cpp methods. I'd prefer `flush` here unless it is added to the OT implementation and this is an override.

Reply:
Fixed in latest